lists/projectDetail.js

```diff
@@ -0,0 +1,33 @@
+const { Text, Url } = require('@keystonejs/fields');
+
+const ProjectDetails = (keystone) => {
+    keystone.createList('project_detail', {
+        fields: {
+            name: {
+                type: Text
+            },
+            project_description: {
```

Comment:
`project_description` must be `product_description` according to the wireframe

![image](https://user-images.githubusercontent.com/53317134/102514217-fa00ce80-406a-11eb-8e4b-a5ac91c4e291.png)